### ISSUE TITLE
Google cloud logging type ERROR for regular training progress output

### ISSUE THREAD

**Issue by danielduckworth** [GoogleCloudPlatform/vertex-ai-samples#242]
## Expected Behavior
Google cloud logging for a custom training job should report training progress as type INFO.

## Actual Behavior
Logs show type ERROR

## Steps to Reproduce the Problem

I've been using the notebook in this repo: [Training, Tuning and Deploying a PyTorch Text Classification Model on Vertex AI](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb)

In the section, "Run Custom Job on Vertex Training with a pre-built container" a package is created and uploaded to a bucket and the job is run. However when monitoring the logs, I get the following:

ERROR   2022-01-26 14:19:34 +1100       workerpool0-0    31%|???       | 5974/19120 [31:21<59:25,  3.69it/s]
ERROR   2022-01-26 14:19:41 +1100       workerpool0-0   Configuration saved in /tmp/xlm-roberta-large/checkpoint-6000/config.json

For some events, it shows:

INFO    2022-01-26 14:19:41 +1100       workerpool0-0   {'loss': 5.3862, 'learning_rate': 1.3723849372384938e-05, 'epoch': 3.14}

**Comment by andrewferlitsch** [comment]
None of these log events are being explicitly logged by the training script. That suggests they are generated on the training service side.

**Comment by kweinmeister** [comment]
From our initial investigation, it looks like these messages are coming from Hugging Face transformers and datasets packages that are writing messages to stderr. The Vertex AI training service is logging them as ERROR accordingly.

**Comment by kweinmeister** [comment]
As this doesn't look to be a problem with the notebook, we will close the issue. Please feel free to reopen if you have more information. Thanks!
